specification/attestation/resource-manager/Microsoft.Attestation/attestation.json

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
Please place these files in a _version_ folder (2018-09-01-preview) in your repo, to identify the API version and allow future versions. To make the review easier, it would be best to copy the files unchanged from this commit to the folder, commit them, and then delete the files in the root as another commit. That way we can diff changes easily during review

Reply:
a new folder 2018-09-01-preview is created, and move the json file to this version